net/socket/socket.c

```diff
@@ -86,6 +86,7 @@ int psock_socket(int domain, int type, int protocol,
 
   psock->s_crefs  = 1;
   psock->s_domain = domain;
+  psock->s_proto = protocol;
```

Comment:
alignment of '='

Reply:
I wonder if we should not set psock->s_type = type here too?  Seems odd to treat it differently.

Reply:
It is done a few lines down from here once the flags are stripped from the argument.

Reply:
> 
> 
> It is done a few lines down from here once the flags are stripped from the argument.

OK.  Good.  hard to see that through the peephole of diff's.